Compilation error

When compiling on ubuntu 20.04 got:
github.com/finogeeks/ligase/syncserver/consumers
../../syncserver/consumers/push.go:199:4: too many arguments in call to func literal
	have (string, *gomatrixserverlib.ClientEvent, string, int, int64, int64, *[]byte, *pushapitypes.PushPubContents, bool)
	want (string, *gomatrixserverlib.ClientEvent, string, int, int64, int64, *[]byte, *pushapitypes.PushPubContents)

> When compiling on ubuntu 20.04 got:
> github.com/finogeeks/ligase/syncserver/consumers
> ../../syncserver/consumers/push.go:199:4: too many arguments in call to func literal
> have (string, *gomatrixserverlib.ClientEvent, string, int, int64, int64, *[]byte, *pushapitypes.PushPubContents, bool)
> want (string, *gomatrixserverlib.ClientEvent, string, int, int64, int64, *[]byte, *pushapitypes.PushPubContents)

Hi, which branch do you use? Thx

--- a/syncserver/consumers/push.go
+++ b/syncserver/consumers/push.go
@@ -193,6 +193,7 @@ func (s *PushConsumer) OnEvent(ctx context.Context, input *gomatrixserverlib.Cli
 			redactOffset int64,
 			eventJson *[]byte,
 			pushContents *push.PushPubContents,
+			isRelatesContent bool,
 		) {
 			s.preProcessPush(ctx, &member, input, &senderDisplayName, memCount, eventOffset, redactOffset, eventJson, pushContents, isRelatesContent)
 			wg.Done()

